src/components/cards/HeadlineCard.tsx

```diff
@@ -77,34 +146,78 @@ export const HeadlineCard: React.FC<Props> = ({ content }) => {
     return (
         <table style={tableStyle}>
```

Comment:
I wonder for this kind of thing whether we can use the `TableRowCell` component and possibly add support for tableStyle and rowStyle (optional) props to help improve concision/avoid all the nesting?

Reply:
I agree we should do this. Both ways of rendering a table are kind of mixed up and used inconsistently throughout the components, so I suggest we tackle this separately and switch to using out Table components as much as possible. I've added a Trello card to track this work: https://trello.com/c/2iyK2nB7/70-review-refactor-usage-of-tables-throughout-to-use-our-table-components-where-possible